templates/pages/paper.jsx

```diff
@@ -42,6 +42,7 @@ var Issue = React.createClass({
         if (response.status >= 400) {
             throw new Error("Bad response from server");
         }
+        alert("Form Submitted Successfully!");
```

Comment:
Hi, you are thinking of making a new page or some sort of status on same
page.
On Mar 12, 2016 10:57 PM, "Abigail Cabunoc Mayes" <notifications@github.com>
wrote:

> In templates/pages/paper.jsx
> <https://github.com/mozillascience/PaperBadger/pull/172#discussion_r55924550>
> :
>
> > @@ -42,6 +42,7 @@ var Issue = React.createClass({
> >          if (response.status >= 400) {
> >              throw new Error("Bad response from server");
> >          }
> > +        alert("Form Submitted Successfully!");
>
> Hi @jinankjain <https://github.com/jinankjain>! I'd rather not use an
> alert here. I was hoping for an update in the interface.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozillascience/PaperBadger/pull/172/files#r55924550>.
>